echo/tests.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+
+C_PROG = "./echo"            
+SHELL_CMD = ["bash", "-c", "echo"]  
+
+
+
+tests = [
+    [""],
+    ["'"],         
+    ["\""],        
+    ["'\"'"],      
+    ["text"],
+    ["-e"],
+    ["-f"],
+    ["-e", "ciao"],
+    ["-e","ciao\\0141"],            
```

Comment:
I suggest to use python's raw strings everywhere, to avoid confusion with double escaping.
You can write something like:
```
["-e", r"ciao\0141"]
```